gulpfile.js

```diff
@@ -33,6 +33,10 @@ gulp.task('move:menu:spy:js', function () {
   return moveJSNodeModule('menuspy/dist/menuspy.js')
 })
 
+gulp.task('move:doc:yall:js', function () {
+  return moveJSNodeModule('yall/dist/yall-2.0.1.min')
```

Comment:
Won't this task break on any update of `yall` to another version? Can we glob this `yall/dist/yall*min`? 

Also, no need to move over minified file, as minification is a later step - always prefer non-minified when copying to source for easier viewing in dev.

Reply:
actually, we have to use dist bc it's ES6, and the minification step does not transpile anything